cmd/bosun/main.go

```diff
@@ -136,10 +136,17 @@ func main() {
 			c.TSDBHost = tsdbHost.Host
 		}
 	}
+	var shortenProxy *url.URL
+	if c.ShortenProxy != "" {
+		shortenProxy, err = url.Parse(c.ShortenProxy)
+		if err != nil {
+			slog.Fatalf("ShortenProxy error: %s", err)
+		}
+	}
 	if *flagQuiet {
 		c.Quiet = true
 	}
-	go func() { slog.Fatal(web.Listen(c.HTTPListen, *flagDev, c.TSDBHost)) }()
+	go func() { slog.Fatal(web.Listen(c.HTTPListen, *flagDev, c.TSDBHost, shortenProxy)) }()
```

Comment:
I don't love the idea of yet another argument to Listen. How about a package variable in web that you set when reading the config above here?